ui/src/components/Log/GoToLineButton.jsx

```diff
@@ -59,11 +59,11 @@ export default class GoToLineButton extends Component {
         <Button
           spanProps={{ className }}
           tooltipProps={{ title: 'Go to Line' }}
-          variant="round"
+          variant="text"
```

Comment:
`text` is the default variant so it's safe to remove this line. Same comment applies to the follow log button.

Reply:
Done.